frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -859,6 +860,10 @@ func downloadArtifactToFile(buildAPI BuildAPI, filename, artifactName, buildID,
 
 // Acloud only works with the first instance: cvd-1
 func runAcloudSetup(runtimesRootDir string) {
+	// Do not run this code in unit tests.
+	if flag.Lookup("test.v") != nil {
```

Comment:
This sort of global state check to enable/disable a feature is not right. Use a config entry or something similar at the call site instead.